packages/itwinui-css/src/header/header.scss

```diff
@@ -11,9 +11,8 @@
   display: flex;
   justify-content: space-between;
   width: 100%;
-  height: $iui-baseline * 5 + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
```

Comment:
![image](https://user-images.githubusercontent.com/37936169/172169888-d6e64ac1-b4d4-4abe-b546-2a48bb713114.png)

Reply:
I talked to Jon and ended up removing the `+ $iui-xxs`. The height of both the default and slim headers are changing here, with the default header now 55px tall (from 57px) and the slim header now 33px tall (from 30px). Now the headers align with the baseline idea and are multiples of 11.